fibers/posix-clocks.scm.in

```diff
@@ -113,15 +128,26 @@
      (+ (* sec #e1e9) nsec))))
 
 (define clock-nanosleep
-  (let* ((ptr (dynamic-pointer "clock_nanosleep" exe))
+  (if is-osx?
+    (let* ((ptr (dynamic-pointer "nanosleep" exe))
+         (proc (pointer->procedure int ptr (list '* '*))))
+    (lambda* (clockid nsec #:key absolute? (buf (nsec->timespec nsec)))
```

Comment:
where is `nsec->timespec` defined?

Reply:
It is defined here https://github.com/Habush/fibers/blob/3c00a8c65555bf93e08372bcbc59b3b42dd7cccc/fibers/posix-clocks.scm.in#L122